scvelo/preprocessing/utils.py

```diff
@@ -502,21 +502,16 @@ def filter_genes_dispersion(
                 with warnings.catch_warnings():  # ignore warning: "Mean of empty slice"
                     warnings.simplefilter("ignore")
                     disp_mad_bin = disp_grouped.apply(robust.mad)
-                val = df["dispersion"].values
                 mu = disp_median_bin[df["mean_bin"].values].values
                 std = disp_mad_bin[df["mean_bin"].values].values
-                df["dispersion_norm"] = np.abs(val - mu) / std
+                df["dispersion_norm"] = ((df["dispersion"] - mu) / std).fillna(0)
```

Comment:
What happened to taking the absolute value of `(df["dispersion"] - mu)`?

Reply:
otherwise it looks good!

Reply:
Yes, sorry - copy and paste error.